app/views/inthenews/wolesoyinkaevents.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1 class="sectiontitle"><%= t "inthenews.title" %></h1>
```

Comment:
would it make sense for this URL structure to be more like `inthenews/wolesoyinka/events` ? and similar below? Or are you not expecting these to be permanent URLs?